test/gtest/uct/test_fence.cc

```diff
@@ -141,6 +157,14 @@ class uct_fence_test : public uct_test {
             }
         }
 
+    static void completion_cb(uct_completion_t *self, ucs_status_t status)
```

Comment:
can you pls also add put operation to fence test? so it would not be just atomics